Kickstarter-iOS/Views/CreatePasswordFooterView.swift

```diff
@@ -0,0 +1,27 @@
+import Prelude
+import Library
+
+internal final class CreatePasswordFooterView: UITableViewHeaderFooterView, NibLoading {
+
+  @IBOutlet private weak var descriptionLabel: UILabel!
+
+  func configure(with email: String?) {
+    guard let email = email else {
```

Comment:
Should this check happen somewhere else maybe? Wouldn't it make reasoning about this UI component a bit simpler if all it took was non-optional string and its sole purpose would be to display it? I'm not sure if a UI component should fail if it's being passed wrong data (it should instead not allow wrong data from being passed to it).

So instead I'd suggest we do `func configure(with email: String) {...}` and handle the optional unwrapping somewhere else.

What do you think Nino?

Reply:
That's how it was before, and I decided to make the string optional so I wouldn't need to provide a default value on the ViewModel 🤦‍♂️ . But what you are saying makes sense and now thinking about it, I guess you're right.